.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set node version to ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install
+        run: npm ci
+
+      - name: Build
+        run: npm run build --if-present
+
+      - name: Test
+        if: github.ref != 'refs/heads/master'
+        run: npm run test
+
+      - name: Test & Coverage
+        if: github.ref == 'refs/heads/master'
+        run: | 
+          npm run test
```

Comment:
Is re-running the test required here? It duplicates the previous step.

Reply:
This will only run when branch is `master`  
Or is `npm run coverage` triggering the test again? :thinking:

Reply:
General idea is good, using the branch reference is more clean and easy way.
It is ok for me, waiting for jerome for further feedbacks but for me is ok

Reply:
It was just nitpicking on coverage usage and test. 
It's ok for me.